README.md

```diff
@@ -1,20 +1,22 @@
 # awesome-custom-images
 
 This list is an attempt to organize and collect public configs of people's custom OStree native containers.
-This will help others build their own and allow for us to learn from each other.  
+This will help others build their own and allow for us to learn from each other.
```

Comment:
Ah, sorry about the unrelated whitespace changes. My vim automatically does that, but I can split them out into a separate commit if you care.